jame3-progressing/progress.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
No, I mean, did you put random input or do you need it to be real data reflecting actual task completion ?